docs/scripts/generate_plugin_docs.py

```
@@ -993,29 +1037,100 @@ def generate_all_docs(self, readme_content: str):
+                continue
+            plugins.extend(self._parse_html_table(section, category))
+        return plugins
+
+    def generate_all_docs(self, readme_content: str):
+        """Generate all labs documentation."""
+        plugins = self.extract_plugins_from_readme(readme_content)
+        if not plugins:
+            logger.warning("No labs features found in README content")
+            return
+
+        logger.info(f"Found {len(plugins)} labs features")
```

Ruff (0.15.10)

[warning] Logging statement uses f-string

(G004)